argus/util.py

```diff
@@ -362,3 +366,57 @@ def get_command(command, command_type=None):
         True: WINDOWS_NANO
     }
 }
+
+
+def get_os_type(instance):
+    """Get the OS type from a `Recipe` or `TestCase` instance."""
+    # pylint: disable=protected-access
+    return instance._backend.remote_client.manager.os_type
```

Comment:
is this the only way to get the OS type? It would be nice to be a public property/getter method